app/src/components/Tabs/Activity/activity-accordion.tsx

```diff
@@ -99,22 +104,24 @@ const ActivityAccordion: FC<ActivityAccordionProps> = ({
                 </Tr>
               </Thead>
               <Tbody>
-                {userActivities?.map((activity: any, i: number) => {
+                {activityData?.map((activity: any, i: number) => {
                   return (
                     <Tr key={i}>
-                      <Td>{activity?.fuelType}</Td>
+                      <Td className="truncate">{activity.id}</Td>
                       <Td>
                         <Tag
                           size="lg"
                           variant="outline"
                           colorScheme="blue"
                           borderRadius="full"
                         >
-                          <TagLabel>{activity?.dataQuality}</TagLabel>
+                          <TagLabel>
+                            {activity?.dataSource.dataQuality}
+                          </TagLabel>
                         </Tag>
                       </Td>
                       <Td>{activity?.fuelConsumption!}</Td>
-                      <Td>{activity?.emissions}</Td>
+                      <Td>{activity?.co2eq}</Td>
```

Comment:
Format units here? (e.g. convert to tons or megatons depending on what the column unit says)